common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -96,7 +92,10 @@ public void onContainerRender(ContainerRenderEvent event) {
                 case TEXTURE -> {
                     int adjustedY;
                     if (displayBottom) {
-                        int displayedTextures = getFilteredEmeraldAmounts(topEmeralds).length;
+                        int displayedTextures = Arrays.stream(getRenderableEmeraldAmounts(topEmeralds))
+                                .filter(s -> !s.equals("0"))
+                                .toArray()
+                                .length;
```

Comment:
Arguably this is weirder than the old way. Checking for equals "0" here and below feels odd.